src/Jupyter/Extensions.cs

```diff
@@ -73,72 +74,88 @@ public static string ToHumanReadableBytes(this long nBytes)
             return $"{nBytes} B";
         }
 
+
         /// <summary>
-        ///     Adds functionality to a given quantum simulator to display
-        ///     diagnostic output with rich Jupyter formatting.
+        ///     An enumerable source of the significant amplitudes of this state
+        ///     vector and their labels, where significance and labels are
+        ///     defined by the values loaded from <paramref name="configurationSource" />.
         /// </summary>
-        /// <param name="simulator">
-        ///     The simulator to be augmented with Jupyter display
-        ///     functionality.
-        /// </param>
-        /// <param name="channel">
-        ///     The Jupyter display channel to be used to display diagnostic
-        ///     output.
-        /// </param>
-        /// <param name="configurationSource">
-        ///      A source of configuration options to be used to set display
-        ///      preferences. Typically, this will be provided by the service
-        ///      provider configured when an execution engine is constructed.
-        /// </param>
-        /// <returns>
-        ///     The value of <paramref name="simulator" />.
-        /// </returns>
-        public static QuantumSimulator WithJupyterDisplay(this QuantumSimulator simulator, IChannel channel, IConfigurationSource configurationSource)
```

Comment:
Really love that with the changes in the other PR, this whole method can now be deleted! ♥